css-canvas-medical/styles.css

```diff
@@ -0,0 +1,241 @@
+.container {
+  max-width: 80%;
+  margin: 10px auto;
+  font-family: "normal", Avenir, sans-serif;
+}
+
+.row {
+  display: flex;
+}
+
+.column-full {
+  flex-basis: 100%;
+  display: flex;
```

Comment:
`column-` classes should not be flexboxes. If you need a `column` to be a flexbox, you likely need a child with the `row` class